css/at-rules/media.json

```diff
@@ -1202,10 +1202,10 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": false
+                "version_added": "61"
```

Comment:
I believe 61 never had a general release (it was only ever a prerelease)—you can bump this to the next version.